pyccel/codegen/printing/fcode.py

```diff
@@ -2820,7 +2817,7 @@ def split_pos_code(line, endpos):
         result = []
         trailing = ' &'
         for line in lines:
-            if len(line)>72 and('"' in line or "'" in line or '!' in line):
+            if len(line)>72 and ('"' in line[72:] or "'" in line[72:] or '!' in line[:72]):
                 result.append(line)
```

Comment:
Can you explain what you are doing here?

Reply:
we don't split a line that contains a string or a comment , where we want to split it

Reply:
Thanks